agent/engine/docker_task_engine.go

```diff
@@ -900,15 +907,28 @@ func (engine *DockerTaskEngine) Capabilities() []string {
 	if engine.cfg.TaskIAMRoleEnabledForNetworkHost {
 		// The "task-iam-role-network-host" capability is supported for docker v1.7.x onwards
 		if _, ok := versions[dockerclient.Version_1_19]; ok {
-			capabilities = append(capabilities, capabilityPrefix+capabilityTaskIAMRoleNetHost)
+			capabilities = appendNameOnlyAttribute(capabilities, capabilityPrefix+capabilityTaskIAMRoleNetHost)
 		} else {
 			seelog.Warn("Task IAM Role for Host Network not enabled due to unsuppported Docker version")
 		}
 	}
 
+	if engine.cfg.TaskENIEnabled {
+		if taskENIAttribute := engine.getTaskENIAttribute(); taskENIAttribute != nil {
+			capabilities = append(capabilities, taskENIAttribute)
+		}
+		if taskENIVersionAttribute := engine.getTaskENIPluginVersionAttribute(); taskENIVersionAttribute != nil {
```

Comment:
Do we still want to register the version if the returned `taskENIAttribute` is nil ?